src/cljs/org/broadinstitute/firecloud_ui/common/components.cljs

```diff
@@ -346,3 +346,48 @@
           sep
           [:span {:style {:verticalAlign "middle"}}
            (:text (last crumbs))]])))})
+
+
+(react/defc SplitPane
+  {:get-initial-state
+   (fn [{:keys [props]}]
+     {:slider-position (or (:initial-slider-position props) 100)})
+   :render
+   (fn [{:keys [props state]}]
+     (let [{:keys [left right top bottom]} props]
+       (assert (or (and left right) (and top bottom)) "Either specify left/right or top/bottom for SplitPane")
+       [:div {:style {:display "flex" :flexDirection (if left "row" "column")}}
+        [:div {:style {:flexGrow 0 :flexShrink 0 :flexBasis (:slider-position @state)}}
+         (or left top)]
+        [:div {:style {:flex "0 0 10px"
```

Comment:
ui tweak: could you put a "grabby" indicator in this divider? Something like the font-awesome bars: http://fontawesome.io/icon/bars/, rotated as appropriate for left/right vs. top/bottom.